src/wallet_new.ts

```diff
@@ -4,12 +4,15 @@ import * as crypto from 'crypto';
 import jwkToPem, { JWK } from 'jwk-to-pem';
 import Arweave from 'arweave';
 import * as mnemonicKeys from 'arweave-mnemonic-keys';
+import { Bytes } from './ardrive';
 
 type PublicKey = string;
 export type ArweaveAddress = string;
 type SeedPhrase = string;
 type TransactionID = string;
-type Winston = string;
+
+/** TODO: Use big int library on Winston types */
```

Comment:
What use cases do you have in mind?

Reply:
Agustin suggested we take this route to preserve accuracy of larger transactions. 

Arweave.js uses `import { BigNumber } from "bignumber.js";` in all of their `ar` class functions, for example.